$OpenBSD: patch-Makefile,v 1.4 2015/01/06 13:19:35 jasper Exp $

- From 6a34745f0e84a990a8fa48e1f1f25be0f08c92a9 Mon Sep 17 00:00:00 2001
  From: Jasper Lievisse Adriaanse <jasper@humppa.nl>
  Date: Sat, 3 Jan 2015 14:30:52 +0100
  Subject: [PATCH] Allow for overriding BIN, SBIN and MAN

- From e4b9c2747bb044f48b35ea02294e73d7c366d9d9 Mon Sep 17 00:00:00 2001
  From: Jasper Lievisse Adriaanse <jasper@humppa.nl>
  Date: Sat, 3 Jan 2015 14:34:50 +0100
  Subject: [PATCH] make the path to vnstat.conf variable

- From ee354ae5e61f332d22b9242f5a2d5af07e4a5373 Mon Sep 17 00:00:00 2001
  From: Jasper Lievisse Adriaanse <jasper@humppa.nl>
  Date: Sun, 4 Jan 2015 22:04:20 +0100
  Subject: [PATCH] Adjust manpages for /var/db/ too, just like vnstat.conf

--- Makefile.orig	Sun Aug 31 12:41:32 2014
+++ Makefile	Sun Jan  4 22:03:02 2015
@@ -4,9 +4,10 @@ SBIN = $(DESTDIR)/usr/sbin
 MAN = $(DESTDIR)/usr/share/man
 
 # bin and man dirs for *BSD
-BIN_BSD = $(DESTDIR)/usr/local/bin
-SBIN_BSD = $(DESTDIR)/usr/local/sbin
-MAN_BSD = $(DESTDIR)/usr/local/man
+BIN_BSD ?= ${PREFIX}/bin
+SBIN_BSD ?= ${PREFIX}/sbin
+MAN_BSD ?= ${PREFIX}/man
+ETC_BSD ?= ${PREFIX}/share/examples/vnstat
 
 .PHONY: vnstat tests check all clean debug install uninstall bsdinstall bsduninstall dist
 
@@ -138,7 +139,6 @@ bsdinstall:
 	fi
 
 # install binaries
-	install -d -m 755 $(DESTDIR)/var/db/vnstat
 	install -s -m 755 src/vnstat $(BIN_BSD)
 	install -s -m 755 src/vnstatd $(SBIN_BSD)
 
@@ -148,26 +148,26 @@ bsdinstall:
 	fi
 
 # install default config if such doesn't exist
-	@if [ ! -f "$(DESTDIR)/etc/vnstat.conf" ]; \
-	then echo "Installing config to $(DESTDIR)/etc/vnstat.conf"; \
-	install -d -m 755 $(DESTDIR)/etc; \
-	install -m 644 cfg/vnstat.conf $(DESTDIR)/etc/vnstat.conf; \
-	sed -e 's/lib/db/g' $(DESTDIR)/etc/vnstat.conf >$(DESTDIR)/etc/vnstat.conf.bsd; \
-	mv -f $(DESTDIR)/etc/vnstat.conf.bsd $(DESTDIR)/etc/vnstat.conf; \
+	@if [ ! -f "$(ETC_BSD)/vnstat.conf" ]; \
+	then echo "Installing config to $(ETC_BSD)/vnstat.conf"; \
+	install -d -m 755 $(ETC_BSD); \
+	install -m 644 cfg/vnstat.conf $(ETC_BSD)/vnstat.conf; \
+	sed -e 's/lib/db/g' $(ETC_BSD)/vnstat.conf >$(ETC_BSD)/vnstat.conf.bsd; \
+	mv -f $(ETC_BSD)/vnstat.conf.bsd $(ETC_BSD)/vnstat.conf; \
 	fi
 
-# update man page	
+# update man page
+	for m in vnstat.1 vnstati.1 vnstatd.1 vnstat.conf.5; do \
+	sed -e 's/lib/db/g' man/$$m > man/$$m.tmp; \
+	mv -f man/$$m.tmp man/$$m; \
+	done
+
 	install -m 644 man/vnstat.1 $(MAN_BSD)/man1
 	install -m 644 man/vnstatd.1 $(MAN_BSD)/man1
 	install -m 644 man/vnstat.conf.5 $(MAN_BSD)/man5
-	gzip -f9 $(MAN_BSD)/man1/vnstat.1
-	gzip -f9 $(MAN_BSD)/man1/vnstatd.1
-	gzip -f9 $(MAN_BSD)/man5/vnstat.conf.5
 	@if [ -f "src/vnstati" ]; \
 	then echo install -m 644 man/vnstati.1 $(MAN_BSD)/man1; \
 	install -m 644 man/vnstati.1 $(MAN_BSD)/man1; \
-	echo gzip -f9 $(MAN_BSD)/man1/vnstati.1; \
-	gzip -f9 $(MAN_BSD)/man1/vnstati.1; \
 	fi
 
 # remove vnstat.conf.1 is such exists in the wrong place
